signals.mml

```diff
@@ -350,7 +351,10 @@ Layer:
             tags->'railway:signal:whistle:states' AS whistle_states,
             tags->'railway:signal:departure:states' AS departure_states,
             tags->'railway:signal:combined:repeated' AS combined_repeated,
-            tags->'railway:signal:main:repeated' AS main_repeated,
+            COALESCE(
+              tags->'railway:signal:main:repeated',
+              tags->'railway:signal:main_repeated'
```

Comment:
So if they are the same we should agree on one tagging and use only that.

Reply:
I will investigate if I can re-tag the existing `railway:signal:main_repeated` tags in OSM :+1:

Reply:
They are different tags, to I split the usage in the query code.